app/src/main/java/com/heig/atmanager/goals/Goal.java

```diff
@@ -22,10 +23,16 @@
  * The app will then generate the goals (cf GoalTodo) for the user to interact with.
  *
  */
-public class Goal {
+public class Goal implements Serializable {
```

Comment:
Careful with Serailizable, the crash during the demo is caused by this #96 , it caused by something that we can serialize
I see on Stackover that Parceble is better (Because it's from Android ), we need more reach to havea conclusion